src/patternfly/components/Nav/nav.scss

```diff
@@ -218,6 +218,21 @@
     display: block;
     visibility: visible;
   }
+
+  // Separator
+  .pf-c-divider {
+    margin-top: var(--pf-c-nav__separator--MarginTop); // change var name at breaking change
+    margin-bottom: var(--pf-c-nav__separator--MarginBottom); // change var name at breaking change
+    margin-left: var(--pf-c-nav__separator--MarginLeft); // change var name at breaking change
+
+    .pf-c-nav__simple-list & {
```

Comment:
This selector compiles to `.pf-c-nav__simple-list .pf-c-nav .pf-c-divider`.

Maybe you should go ahead and create vars for `--pf-c-nav__c-divider--MarginTop/Bottom/Left`, then modify the vars in the CSS blocks for `.pf-c-nav__simple-list` and `.pf-c-nav__item`?

Reply:
Yep I updated all the components to be like this now, and then we can just remove vars at breaking change rather than changing var names etc.

Reply:
Still seeing the problem with the nav selectors. https://www.sassmeister.com/gist/966ff367643b853eea5b2dc4a35e5d09

It works with `.pf-c-nav__separator` because that element is outside of the scope of `.pf-c-nav`

Reply:
Oh sorry I didn't understand that the first time. Could you check that they're in the right blocks I wasn't 100% sure @mcoker